types/react-native-fbsdk/index.d.ts

```diff
@@ -0,0 +1,1090 @@
+// Type definitions for react-native-fbsdk 0.6
+// Project: https://github.com/facebook/react-native-fbsdk
+// Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { ComponentClass, Component } from 'react';
+import { StyleSheet } from 'react-native';
+
+// tslint:disable-next-line:export-just-namespace
+export = FBSDK;
+export as namespace FBSDK;
```

Comment:
You just shouldn't need line 12 at all. It only modifies what shows up in non-module files